### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by hoodmane**
Left out:
* opencv-python
* matplotlib
@ryanking13 maybe you can work on those?

### CODE REVIEW

**Comment by ryanking13** [comment]
> * @ryanking13 maybe you can work on those?

Sure I will.

File: packages/msprime/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-  sha256: e840bed2ba37ae572b234f0b55af006dc408881593f184304481ee6058ea6f30
+  url: https://files.pythonhosted.org/packages/e6/81/8b16831a9e58151da55fd63e13d052ba27ea647022b6555964115a38c6cd/msprime-1.2.0.tar.gz
+  sha256: 60025ad9fd30d827a72ae6e760b6cff07a1d0e169b2c1da1024cb0fc23e4a7aa
 build:
   script:
-    export LIBGSL_INCLUDE_PATH=$(pkg-config --cflags-only-I --dont-define-prefix gsl)
-    export LIBGSL_LIBRARY_PATH=$(pkg-config --libs-only-L --dont-define-prefix gsl)
+    export LIBGSL_INCLUDE_PATH=$(pkg-config --cflags-only-I --dont-define-prefix
+    gsl) export LIBGSL_LIBRARY_PATH=$(pkg-config --libs-only-L --dont-define-prefix
+    gsl)
```
Comment (ryanking13) [inline packages/msprime/meta.yaml:11]: I guess this should be `script: |`.
Not sure why this wasn't throwing an error previously.

**Comment by rth** [comment]
Though it looks like the matplotlib build fails even without changing the verstion.

**Comment by ryanking13** [comment]
> Though it looks like the matplotlib build fails even without changing the verstion.

Yeah it was failing also in main. #2760 will fix.

**Pull request merged**